### DISCUSSION TITLE
Format capacity

### DISCUSSION THREAD

**Discussion by yuchen-sun** [juicedata/juicefs#874] [Q&A] [answered]
If capacity is specified during formatting, is there an opportunity to modify it again later?
Sorry, I can't find the documentation

**Comment by davies** [comment, marked as answer]
Yes, you can use format to update the capacity, other settings should be provided as options in the same time.

**Reply by yuchen-sun** [reply]
btw：Is multi-tenancy in your roadmap ?  : )

**Reply by davies** [reply]
We don't have a clear picture of what multi-tenancy should be, so it's not in the roadmap yet.

We'd like to hear more on this from community.
